usagov/usagov-2021
# USAGOV-2020-CEO-A11Y: Update _uswds-theme-custom-styles.scss

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-2020

## Description

Add padding between error bars and ceo fields. 

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

- At /elected-officials click Find my elected officials (or at es/funcionarios-electos click Encontrar a mis funcionarios electos) 
- When the error bar shows on the left make sure there is 1rem gap
- See these photos for example: 

![after](https://github.com/user-attachments/assets/71bbfffc-4b71-4b12-a20f-e34ffeb5c6bb)
![before](https://github.com/user-attachments/assets/09c525bc-f432-4f12-b94c-05d0f60a7d11)

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
